addons/arcadian/base/config_turret_gatling.hpp

```diff
@@ -0,0 +1,47 @@
+class Turrets: Turrets {
+    class MainTurret: MainTurret {
+        body = "mainTurret";
+        gun = "mainGun";
+        viewGunnerInExternal = 1;
+        minElev = -10;
+        maxElev = 10;
+        initElev = 0;
+        soundServo[] = {"", db-40, 1};
```

Comment:
Where is `db` defined?

Reply:
Far as i'm aware it's something you can use with sounds instead of setting a number, game automatically uses it.

Reply:
`"db-40"` should be quoted it seems. It's evaluated similar to code.

Reply:
Uses the same dummy sound as Offroad Armed does, volume is 0 now.